index.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Portfolio page</title>
+	<meta name="description" content="Holly's portfolio page">
+	<meta name="author" content="Holly Bedford">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css" type="text/css">
```

Comment:
Make sure you add the link to the normalize file after your 'style.css' so none of your styles are overridden!